src/cow-react/modules/wallet/web3-react/connection/index.tsx

```diff
@@ -107,10 +113,16 @@ export function ConnectWalletOptions({ tryActivation }: { tryActivation: TryActi
   const walletConnectionOption =
     (!isInjectedMobileBrowser && <WalletConnectOption tryActivation={tryActivation} />) ?? null
 
+  // Wallet-connect based
   const zengoOption = (!isInjectedMobileBrowser && <ZengoOption tryActivation={tryActivation} />) ?? null
   const ambireOption = (!isInjectedMobileBrowser && <AmbireOption tryActivation={tryActivation} />) ?? null
   const alphaOption = (!isInjectedMobileBrowser && <AlphaOption tryActivation={tryActivation} />) ?? null
 
+  // Injected
+  const tallyOption =
+    (!isInjectedMobileBrowser && isChrome && !isChromeMobile && <TallyWalletOption tryActivation={tryActivation} />) ??
```

Comment:
this is a non trivial/obvious condition, can you add a comment how do you arrive to this?

Reply:
What i mean, can you add a comment with your intent? 

i.e. Just an example, i don't know if I got it right
// Show tally option only in chrome (includes Brave too), but not for ..

Reply:
Updated